Mods/ModVarietyPack/Defs/ThingDefs/Plants_Cultivated.xml

```diff
@@ -61,8 +61,8 @@
       <soundHarvesting>Harvest_Standard</soundHarvesting>
       <soundHarvestFinish>Harvest_Standard_Finish</soundHarvestFinish>
       <sowWork>150</sowWork>
-      <!--<fertilityMin>0.5</fertilityMin>
-      <fertilityFactorGrowthRate>1.0</fertilityFactorGrowthRate>-->
+      <fertilityMin>0.5</fertilityMin>
+      <fertilityFactorGrowthRate>1.0</fertilityFactorGrowthRate>
```

Comment:
I am working on it. 
Did you try it with 1.12? 
Also playing in temperate, I felt it was much better. For you too? Or just as bad as before?